### DISCUSSION TITLE
Drag-from-outside-listener?

### DISCUSSION THREAD

**Discussion by Reached** [codex-team/editor.js#1914] [Q&A]
Hi there, love the project!

I would like to be able to drag in a div containing a block of text from outside editorjs (html5 drag/drop?), and having it inserted at the current caret position, is it even possible to write a plugin to do this, and are there any API's you could point me to? 

Thanks in advance!

**Comment by Aarbel** [comment]
@Reached have you found more infos about this ?

**Comment by Reached** [comment]
@Aarbel good question, I think I ended up using TipTap instead, and then wrote my own custom thing to do it :/
